source/views/Extension/Views/ImportPrivateKey/Steps/Step2.jsx

```diff
@@ -0,0 +1,118 @@
+import { Container, Button, TextInput, FormItem, UserIcon } from "@components";
+import Grid from "@material-ui/core/Grid";
+import useStyles from "../styles";
+import React, { useState } from "react";
+import { useRouter } from "@components/Router";
+import { HANDLER_TYPES, sendMessage } from "@background/Keyring";
+import Picker from "emoji-picker-react";
+import { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+
+const Step2 = ({ privateKey }) => {
+  const { t } = useTranslation();
+
+  const [loading, setLoading] = useState(false);
+  const [disabled, setDisabled] = useState(false);
```

Comment:
We should invert the `disabled` value logic, if `true` the button should be disabled